rust/tools/qltest.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash
 
 set -eu
+set -o pipefail
 
 export RUST_BACKTRACE=full
 QLTEST_LOG="$CODEQL_EXTRACTOR_RUST_LOG_DIR"/qltest.log
-if ! "$CODEQL_EXTRACTOR_RUST_ROOT/tools/$CODEQL_PLATFORM/extractor" --qltest >> "$QLTEST_LOG" 2>&1; then
-  cat "$QLTEST_LOG"
+TMP_OUT="$(mktemp)"
```

Comment:
Perhaps use CODEQL_EXTRACTOR_RUST_SCRATCH_DIR ? That's easier to port to `qltest.cmd` too.

Reply:
ah, right, I forgot about `qltest.cmd`...

Reply:
I'm not sure this color removal is required on Windows, so I'd leave that alone until I try that. But I did apply the `progress+` thing there as well now.